charts/redis/templates/DeploymentConfig.yaml

```diff
@@ -25,7 +25,7 @@ spec:
           valueFrom:
             secretKeyRef:
               key: redis_password
-              name: "{{ .Values.sealed_secrets.sealed_secret_name }}-incluster-secret"
+              name: "{{ .Values.sealed_secrets.sealed_secret_name }}"
```

Comment:
Will this affect any existing application?

Reply:
No @gkmahendran09. I've bumped the version. So all applications are using the `1.0.3` currently and this will be a new version`1.0.4`.